package.json

```diff
@@ -32,6 +32,7 @@
         "fela-plugin-unit": "^11.6.0",
         "fela-plugin-validator": "^11.6.0",
         "firebase": "^8.9.1",
+        "gitmoji-cli": "^4.5.0",
```

Comment:
This is not needed here, install it better globally

Reply:
Already did in upcoming branch.